Developer.md

```diff
@@ -0,0 +1,22 @@
+# Building flyxc
+
+## Preparation
+
+Copy `keys.ts.dist` to `keys.ts` and fill it out (there are two such files)
+
+## Installation of dependencies
+`npm install --no-fund --no-audit` To install all dependencies (possible to use also yarn: `yarn install`)
+
+
+## Building flyxc
+
+`npx nx run-many -t build`
+
+## Starting server
+
+`npm run dev`
```

Comment:
`npx nx dev` is better.

(Note that you can install nx globally with `npm i -g nx` so that you don't have to use `npx`)